ipfs-paste

```diff
@@ -64,25 +73,38 @@ fi
 if [ $paste ]; then
   test -n "$clipaste" || die "clipboard paste unkown for $(uname)"
   log "paste to ipfs..."
-  fhash=$($clipaste | ipfs add -q)
+
+  date=$(date +"%Y-%m-%dT%H:%M:%SZ")
+  fpath=$(mktemp "/tmp/paste.$date.XXXXXX.txt") ||
+  die "could not 'ktemp /tmp/paste.$date.XXXXXX.txt'"
```

Comment:
ok sounds good, no worries. (i do think it is confusing for the reader, the non-indentation makes it look like it's in the same scope, not within a conditional. but no use fighting everyone's sh indentation scripts